src/Tribe/RSVP.php

```diff
@@ -61,6 +61,13 @@ class Tribe__Tickets__RSVP extends Tribe__Tickets__Tickets {
 	public $security_code = '_tribe_rsvp_security_code';
 
 	/**
+	 * Meta key that if this attendee wants to show on the attendee list
+	 *
+	 * @var string
+	 */
+	public $optout = '_tribe_rsvp_attendee_optout';
```

Comment:
We've got two separate words here - maybe add in an underscore them so we have `$opt_out` instead of `$optout`? Though the meaning is obvious now I'm writing this comment, it did give me a moment of confusion on first reading.